src/main.cpp

```diff
@@ -105,6 +107,28 @@ bool testSPIRAM(void) {
 	}
 #endif
 
+// Get next RFID-tag if already set
+void loadNextRfid(void) {
+	if (recoverRfidFromBluetooth) {
```

Comment:
Instead of nesting the whole code block in an if condition, better do:

```c
if (!recoverRfidFromBluetooth) {
    return;
}
recoverRfidFromBluetooth = false; // only try it once per bootup
[...]
```

Reply:
Much nicer 👍

Reply:
@Joe91 It's your PR, so I would suggest you do it :wink: